src/data.jsx

```diff
@@ -0,0 +1,295 @@
+import React from 'react';
+import { SiOpenaigym } from 'react-icons/si';
+import avatar1 from './images/avatar1.jpg';
+import avatar2 from './images/avatar2.jpg';
+import avatar3 from './images/avatar3.jpg';
+import avatar4 from './images/avatar4.jpg';
+import avatar5 from './images/avatar5.jpg';
+
+export const links = [
+  {
+    name: 'Home',
+    path: '/',
+    id: 'home',
+  },
```

Comment:
To not revert my previous PR:
```suggestion
  {
    name: 'Join Us',
    path: '/joinus',
    id: 'joinus',
  },
```